package.json

```diff
@@ -81,10 +82,10 @@
     "link-cli": "yarn run create-agoric-cli",
     "create-agoric-cli": "node ./scripts/create-agoric-cli.cjs",
     "format": "yarn prettier --write 'packages/**/*.{js,jsx,ts,tsx}' 'golang/**/*.{js,jsx,ts,tsx}'",
```

Comment:
Should we rename to `lint-fix:format` in the principle of consistency? And introduce a `lint-fix` that performs the workspace the above and a workspace `lint-fix` (for the eslint fix)

Reply:
I've been burned by `eslint --fix` enough not to trust it so I'd personally shy away from using a catch-all `yarn lint-fix` but I wouldn't object to a PR making that change.

Reply:
Agreed! But I believe it's used by some. I personally relies on my editor for formatting auto fixes, so I use neither. I'll let others chime in.